pom.xml

```diff
@@ -143,31 +143,31 @@
     <easymock.version>3.4</easymock.version>
     <objenesis.version>2.6</objenesis.version>
 
-    <cloud.version>0.21.1-alpha-SNAPSHOT</cloud.version>
+    <cloud.version>0.21.1-alpha</cloud.version>
     <bigquery.version>0.21.1-beta-SNAPSHOT</bigquery.version>
-    <compute.version>0.21.1-alpha-SNAPSHOT</compute.version>
-    <contrib.version>0.21.1-alpha-SNAPSHOT</contrib.version>
+    <compute.version>0.21.1-alpha</compute.version>
```

Comment:
I don't think that alpha should move a version without beta also moving a version. Also, logging & pubsub wouldn't pick up the corresponding dependency updates without updating them too (logging implies GA version update). I'd like to stick to patch bumps across everything instead of just across some version tracks.